packages/zk-certificates/contracts/GalacticaTwitterSBT.sol

```diff
@@ -44,6 +47,25 @@ contract GalacticaTwitterSBT is ERC721, AccessControl {
         _mint(user, newNFTId);
     }
 
+    /**
+     * @dev Mint a NFT for a user with customized metadata
+     * @param user Address that should receive the NFT
+     * @param uri URI for the NFT
+     * @param name Name for the NFT
+     * @param symbol Symbol for the NFT
+     */
+    function mint(address user, string memory uri, string memory name, string memory symbol) public onlyRole(ISSUER_ROLE) {
```

Comment:
Would be nice to have a unittest for such contracts. Just to make sure they work and the quality stays the same when doing future changes.

Reply:
In the end Nick wants to keep this contract aside and returns to deploy multiple contracts with different metadata.